### ISSUE TITLE
Cannot start AIO properly - nextcloud-aio-apache has unhealthy status

### ISSUE THREAD

**Issue by vawaver** [nextcloud/all-in-one#1664]
After some weeks I started my Nextcloud AIO v 3.1.0 to upgrade.
But my Nextcloud instance does not work.

So I try to find where is a problem.

All containers are running as healthy status in my Portainer. Only nextcloud-aio-apache has unhealthy status. (Screenshots attached).
I checked the logs and there are some errors I can understand. Logs from containers are attached.

AIO is stuck now and I am not able to stop containers from the AIO web interface and cannot be updated to the latest version.
How to solve the problem?

#### Current channel  latest
[_nextcloud-aio-apache_logs.txt](https://github.com/nextcloud/all-in-one/files/10343230/_nextcloud-aio-apache_logs.txt)
[_nextcloud-aio-collabora_logs.txt](https://github.com/nextcloud/all-in-one/files/10343231/_nextcloud-aio-collabora_logs.txt)
[_nextcloud-aio-nextcloud_logs.txt](https://github.com/nextcloud/all-in-one/files/10343233/_nextcloud-aio-nextcloud_logs.txt)
![aio](https://user-images.githubusercontent.com/52236863/210535590-7507140a-4ac0-413a-b97a-6b2808625f3b.png)
![portainer](https://user-images.githubusercontent.com/52236863/210535597-74e4fbf7-81af-48b1-8fb3-7a787fc3a612.png)

**Comment by szaimen** [comment]
Hi, I am really not sure why the apache container does not work for you. Can you try to stop apache with `sudo docker stop nextcloud-aio-apache` and reload the aio interface? This should allow you to stop your containers again and hopefully allow you to upgrade afterwards.

**Comment by vawaver** [comment]
I updated my AIO to 4.1.0 but Nextcloud does not start.
nextcloud-aio-apache has still unhealthy status.
nextcloud-aio-talk - not started

What should I do?
Is there some way how to recreate it all without losing data?

Sending my screenshots and logs from the containers

![aio01](https://user-images.githubusercontent.com/52236863/210560547-e9ad58b2-800c-4ae2-8e24-178f15b02da7.png)
![portainer01](https://user-images.githubusercontent.com/52236863/210560550-53a20fff-e599-4e16-b4e4-2036dc51c11d.png)


[_nextcloud-aio-apache_logs(1).txt](https://github.com/nextcloud/all-in-one/files/10344259/_nextcloud-aio-apache_logs.1.txt)
[_nextcloud-aio-nextcloud_logs(1).txt](https://github.com/nextcloud/all-in-one/files/10344261/_nextcloud-aio-nextcloud_logs.1.txt)

**Comment by Kirk1984** [comment]
Isn't the problem that nextcloud isn't starting because of the "502 Bad Gateway" when checking the domain?

**Comment by szaimen** [comment]
> Isn't the problem that nextcloud isn't starting correctly because of the "502 Bad Gateway" when checking the domain?

No,, I don't think so. The apache container does not start because the nextcloud container is somehow not starting corrrectly. But nothing useful is in the logs currently. Hence I asked for new logs of the nextcloud container.

**Comment by szaimen** [comment]
Also ut would be great if you could post the mastercontainer logs @vawaver

**Comment by vawaver** [comment]
Hello,

I am sending the requested logs.

Hope it helps.

[_nextcloud-aio-apache_logs.txt](https://github.com/nextcloud/all-in-one/files/10344584/_nextcloud-aio-apache_logs.txt)
[_nextcloud-aio-collabora_logs.txt](https://github.com/nextcloud/all-in-one/files/10344585/_nextcloud-aio-collabora_logs.txt)
[_nextcloud-aio-mastercontainer_logs.txt](https://github.com/nextcloud/all-in-one/files/10344586/_nextcloud-aio-mastercontainer_logs.txt)
[_nextcloud-aio-nextcloud_logs.txt](https://github.com/nextcloud/all-in-one/files/10344587/_nextcloud-aio-nextcloud_logs.txt)
[_nextcloud-aio-talk_logs.txt](https://github.com/nextcloud/all-in-one/files/10344588/_nextcloud-aio-talk_logs.txt)

**Comment by szaimen** [comment]
Hm... so it indeed looks like and issue with docker networking since the apache is seemingly not knowing the domain of `nextcloud-aio-nextcloud`. 

Did you maybe update your unraid server software after which the problem appeared?

Also, can you post the output of `sudo docker inspect nextcloud-aio-apache`?

**Comment by vawaver** [comment]
I do not use Unraid.
Sending the output as you request.

[output_log.txt](https://github.com/nextcloud/all-in-one/files/10344684/output_log.txt)

**Comment by szaimen** [comment]
> I do not use Unraid.

Sorry, my fault.

**Comment by vawaver** [comment]
> > I do not use Unraid.
> 
> Sorry, my fault.

Not a problem.
I do not know what happened because I did not changed any settings on my server.

**Comment by szaimen** [comment]
So I think we have found the issue. Somehow the apache container seem to be not connected to the nextcloud-aio network. 

Can you also send the output of `docker network inspect nextcloud-aio` ?

**Comment by vawaver** [comment]
> docker network inspect nextcloud-aio

Hello,
I am sending the output.
[network_aio.txt](https://github.com/nextcloud/all-in-one/files/10349875/network_aio.txt)

@Kirk1984  - I run the AIO in the same way as my default installation of version 3.1.0 . No change.

**Comment by lucianomilani** [comment]
In the clean install APACHE container and Nextcloud Container dont install (red flag in scream) i trie a lot times in proxmox envoriment

**Comment by szaimen** [comment]
Hi @lucianomilani this is unrelated to this issue. Please open a new discussion for you problem. Thanks!
See https://github.com/nextcloud/all-in-one/discussions/new?category=questions

**Comment by szaimen** [comment]
> > docker network inspect nextcloud-aio
> 
> Hello, I am sending the output. [network_aio.txt](https://github.com/nextcloud/all-in-one/files/10349875/network_aio.txt)

@vawaver thanks! Can you check if that makes it work?
`sudo docker network connect nextcloud-aio nextcloud-aio-apache`

**Comment by vawaver** [comment]
> `sudo docker network connect nextcloud-aio nextcloud-aio-apache`

I run the command in the console without any output and I think it helped because nextcloud-aio-apache is running with a "healthy" status in Portainer and my Nextcloud page is finally available now.

Sending logs as attachment.

[_nextcloud-aio-apache_logs(1).txt](https://github.com/nextcloud/all-in-one/files/10352007/_nextcloud-aio-apache_logs.1.txt)

**Comment by szaimen** [comment]
So when you now restart all containers from the AIO interface. Does the error re-appear?

**Comment by vawaver** [comment]
> So when you now restart all containers from the AIO interface, does the error re-appear?

I rebooted the whole linux server and it seems it works fine finally.
Apache and Nextcloud dockers started normally.
I suppose the problem is solved.

**Comment by szaimen** [comment]
Okay, moving to discussions then. Feel free to reply if it should happen again!
